Python/pylifecycle.c

```diff
@@ -471,11 +600,29 @@ _Py_InitializeEx_Private(int install_sigs, int install_importlib)
         Py_XDECREF(warnings_module);
     }
 
+    _Py_Initialized = 1;
+
     if (!Py_NoSiteFlag)
         initsite(); /* Module site */
+
+	return 0;
 }
 
 void
+_Py_InitializeEx_Private(int install_sigs, int install_importlib)
+{
+    _PyCoreConfig core_config = _PyCoreConfig_INIT;
+
+    /* TODO: Moar config options! */
+    core_config.ignore_environment = Py_IgnoreEnvironmentFlag;
+    core_config._disable_importlib = !install_importlib;
+    _Py_InitializeCore(&core_config);
+    _Py_InitializeMainInterpreter(install_sigs);
```

Comment:
But for anyone just reading these comments: "install_sigs" becomes the first field in the new config struct for the main interpreter, rather than being needed to get the core runtime going :)